package.json

```diff
@@ -11,7 +11,7 @@
     "travis-deps": "pip install -t lib -r requirements.travis.txt --upgrade",
     "dev-deps": "pip install -r requirements.dev.txt --upgrade",
     "test": "python -m unittest discover -p *_test.py  -b",
-    "coverage": "python -m coverage erase && python -m coverage run -m unittest discover -p *_test.py -s tests -b && python -m coverage html",
+    "coverage": "python -m pip install --no-deps -r requirements.dev.txt && python -m coverage erase && python -m coverage run -m unittest discover -p *_test.py -s tests -b && python -m coverage html",
```

Comment:
I think you need to remove the `-s tests` because we no longer have a tests directory.

Also, today I landed a change that uses `-p "*_tests.py"` in quotes to avoid shell wildcard expansion if there happens to be a match in the current directory.  So, you just need to have that same change.

Reply:
@jrobbins I have made the above-mentioned changes